jupyterhub/scopes.py

```diff
@@ -36,13 +36,13 @@
             'read:users:name',
             'read:users:groups',
             'read:users:activity',
-            'read:users:roles',
         ],
+        # TODO: add read:users:admin and read:users:roles as subscopes here once implemented
     },
     'read:users:name': {'description': 'Read-only access to users’ names.'},
     'read:users:groups': {'description': 'Read-only access to users’ group names.'},
-    'read:users:roles': {'description': 'Read-only access to users’ role names.'},
     'read:users:activity': {'description': 'Read-only access to users’ last activity.'},
+    # TODO: add read:users:admin and read:users:roles once implemented
```

Comment:
Is `read:users:admin` still part of the scopes we use?

Reply:
Ah, I think it's only used within the identify scopes and nothing else so we probably don't need to add it to the scope definitions, thanks!